tox.ini

```diff
@@ -1,7 +1,10 @@
 #
 # Configuration for tox and pytest
 
+# first five ignores are defaults per http://flake8.readthedocs.io/en/latest/config.html#default
 [flake8]
+ignore = E123, E133, E226, E241, E242, W503, E402
+max-line-length = 77
 exclude = dist,docs,*.egg-info,.git,ref,_scratch,.tox
```

Comment:
I just quickly created that file with header via the Word Online interface. So it has whatever strangeness came from that! We should probably replace it with a "purer" docx with just the header.

S1HP1 sounds good to me!